api/plugins/ginfo/ginfo.py

```diff
@@ -0,0 +1,153 @@
+
+"""
+  Integration into the Ginfo Map via Firebase REST API
+"""
+from django.http import HttpResponse, HttpRequest
+from django.db.models import Q
+import requests
+import math
+from api.models import Character
+from ginfocharacter import GinfoCharacter
+from datetime import datetime
+from firebase_pushid import PushID
+
+
+class GinfoPlugin(object):
+    """
+      GinfoPlugin provides methods to post a position to the ginfo firebase database
+    """
+    API_URL = 'https://ginfo-34baa.firebaseio.com'
+    GAME = 'conanexiles'
+    MAP = 'conanexiles_desert'
+    GINFO_USER_UID = 'y67Efa3kddSuhb26UoUapIJakMh1'
```

Comment:
Yeah I see, the name of the constant does not imply that its a specific user, I'll change that!